bigchaindb/common/transaction.py

```diff
@@ -421,7 +420,7 @@ class Transaction(object):
     TRANSFER = 'TRANSFER'
     GENESIS = 'GENESIS'
     ALLOWED_OPERATIONS = (CREATE, TRANSFER, GENESIS)
-    VERSION = '.'.join(bigchaindb.version.__short_version__.split('.')[:2])
+    VERSION = '1.0'
```

Comment:
Up until now the transactions had the same version as the server.
This is saying that the transaction version and the server version should be two different things.
I don't necessarily disagree but this is a decision that affects all bigchaindb drivers